packages/lab/src/Flow/Node/Node.tsx

```diff
@@ -144,11 +160,28 @@ export const HvFlowNode = ({
     >
       {(subtitle || actions) && (
         <div className={classes.subtitleContainer}>
-          {subtitle && (
-            <div>
+          {subtitle &&
+            (disableInlineEdit ? (
               <HvTypography>{subtitle}</HvTypography>
-            </div>
-          )}
+            ) : (
+              <HvInlineEditor
+                defaultValue={subtitle}
+                showIcon
+                classes={{
+                  root: css({
+                    display: "inline-flex",
+                    flexGrow: 1,
+                  }),
+                  button: css({ justifyContent: "space-between" }),
+                  inputRoot: css({
+                    width: inlineEditorWidth,
+                  }),
+                }}
+                onBlur={(evt, value) =>
+                  setNodeData((prev) => ({ ...prev, label: value }))
```

Comment:
I was thinking if it wouldn't be better to use a different name than `label` since it seems generic enough that some users may already be using it. 
Maybe `customNodeLabel` or other.